### PR TITLE
[updates/issue] fixing styling issues from rm planifolia 

### PR DESCRIPTION
**Pull request by phillimorland**
partial fix for #2103

### CODE REVIEW

**Comment by phillimorland** [comment]
This is another interim step as will have meeting with Marcelo tomorrow to discuss which styles should be kept and which should be done away with, still not sure if extending bootstrap btns was way to go rather then re-writting the old mixin but guess time will tell :(

File: apps/documents/assets/ChapterNavItem.jsx [hunk 1]
```diff
@@ -17,7 +16,7 @@ const ChapterNavItem = (props) => {
     <div className="row" data-testid="chapter-nav-item">
       <button
         type="button"
-        className={classNames('commenting__content', 'commenting--toc__button', 'btn btn--light', 'btn--small', { active: props.active })}
+        className={(props.active && 'active') + 'commenting__content commenting--toc__button btn btn--light btn--small'}
```
Comment (fuzzylogic2000) [inline apps/documents/assets/ChapterNavItem.jsx]: is it fine like this or do we need a space between `active` and `commenting content`?

Comment (fuzzylogic2000) [reply apps/documents/assets/ChapterNavItem.jsx]: But cool, nice removal of package!

File: adhocracy-plus/assets/scss/components/_button.scss [hunk 2]
```diff
@@ -4,103 +4,136 @@
...
 
 .btn {
     @extend %button-base;
-
-    text-decoration: none;
 }
 
-.btn--light {
-    @extend .btn-outline-light;
+.btn.btn--primary {
```
Comment (fuzzylogic2000) [inline adhocracy-plus/assets/scss/components/_button.scss:14]: Why do these all have the btn and the variant? To make it more specific and overwrite other styles?

**Review by fuzzylogic2000 — COMMENTED** [review]
Somehow, there was a diff outdated and I couldn't add a comment, so here some questions already and then I'll go on reviewing!

File: adhocracy-plus/assets/scss/components/_button.scss (hunk 2 above)
Comment (phillimorland) [reply adhocracy-plus/assets/scss/components/_button.scss:14]: @fuzzylogic2000 it was to make it more specific so we could still specify styling for just the `btn`class but it actually didn't work as planned, and i'm not sure if we should have anything more then basic styling assigned to btn anyway and as that was the black btn and given that it is difficult to show different states i'm not sure what to do for that one, once this is merged i'll let Marcelo know and we are going to have a chat about it tomorrow

**Comment by phillimorland** [comment]
> Somehow, there was a diff outdated and I couldn't add a comment, so here some questions already and then I'll go on reviewing!

sorry no one was assigned so thought it wasn't being reviewed and pushed some updates

File: adhocracy-plus/assets/scss/components/_button.scss [hunk 3]
```diff
@@ -4,103 +4,136 @@
...
     &:hover,
     &:active,
-    &:disabled,
-    &.is-disabled {
-        box-shadow: none;
-        background-color: $gray-lightest;
+    &:focus-within,
+    &:focus-visible {
+        background-color: $brand-tertiary !important;
+        color: invert($print-black) !important;
```
Comment (fuzzylogic2000) [inline adhocracy-plus/assets/scss/components/_button.scss:72]: Why `invert($print-black)` instead of using some variable for white?

**Review by fuzzylogic2000 — APPROVED** [review]
Cool, many improvements! And one more question... :)

**Comment by fuzzylogic2000** [comment]
> > Somehow, there was a diff outdated and I couldn't add a comment, so here some questions already and then I'll go on reviewing!
> 
> sorry no one was assigned so thought it wasn't being reviewed and pushed some updates

Right, sorry!

File: apps/documents/assets/ChapterNavItem.jsx (hunk 1 above)
Comment (fuzzylogic2000) [reply apps/documents/assets/ChapterNavItem.jsx]: yes, found it! :)

File: adhocracy-plus/assets/scss/components/_button.scss (hunk 2 above)
Comment (fuzzylogic2000) [reply adhocracy-plus/assets/scss/components/_button.scss:14]: Cool, let's merge then!

**Pull request merged**